migrations/migrations/4.js

```diff
@@ -78,6 +105,7 @@ const notes = `
 - Automatically rename **variable names** and **ordinal/categorical values** to meet stricter requirements. Only letters, numbers, and the symbols \`.\`, \`_\`, \`-\`, \`:\` will be permitted. Spaces will be replaced with underscore characters (\`_\`), and any other symbols will be removed. Variables that meet these requirements already **will not be modified**.
 - Add a numerical suffix (\`variable1\`, \`variable2\`, etc.) to any variables or categorical/ordinal values that clash as a result of these changes.
 - Rename node and edge types to ensure they are unique, and conform to the same requirements as variable names. Names that clash will get a numerical suffix, as above.
+- Removes any non-boolean additional attributes from prompts.
```

Comment:
Can we put this point below the one after it, and replace the text with the following:

"Removes any non-boolean 'additional variables' from prompts. It was neccesary to simplify this feature, and so only boolean variable types will be supported moving forwards. Any non-boolean variables you created that will be removed by this migration will remain in your codebook, but will be marked 'unused'. You should review and remove this manually, or replace them with equivalent boolean variables.

Reply:
Sounds good!